### ISSUE TITLE
Redis & PHP on Ubuntu 14.04 x32

### ISSUE THREAD

**Issue by alextyrey** [puphpet/puphpet#1291]
```
==> default: Error: Execution of '/usr/bin/apt-get -q -y -o DPkg::Options::=--force-confold install php5-redis' returned 100: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: Some packages could not be installed. This may mean that you have
==> default: requested an impossible situation or if you are using the unstable
==> default: distribution that some required packages have not yet been created
==> default: or been moved out of Incoming.
==> default: The following information may help to resolve the situation:
==> default: 
==> default: The following packages have unmet dependencies:
==> default:  php5-redis : Depends: phpapi-20131226+lfs but it is not installable
==> default: E: Unable to correct problems, you have held broken packages.
==> default: 
==> default: Error: /Stage[main]/Main/Puphpet::Php::Pecl[redis]/Package[php5-redis]/ensure: change from purged to present failed: Execution of '/usr/bin/apt-get -q -y -o DPkg::Options::=--force-confold install php5-redis' returned 100: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: Some packages could not be installed. This may mean that you have
==> default: requested an impossible situation or if you are using the unstable
==> default: distribution that some required packages have not yet been created
==> default: or been moved out of Incoming.
==> default: The following information may help to resolve the situation:
==> default: 
==> default: The following packages have unmet dependencies:
==> default:  php5-redis : Depends: phpapi-20131226+lfs but it is not installable
==> default: E: Unable to correct problems, you have held broken packages.
==> default: 
```

**Comment by CWSpear** [comment]
I have that exact error with a similar `config.yaml`: https://gist.github.com/CWSpear/009b248a1a3094c299da

**Comment by qutwala** [comment]
The same to me. Is there any workaround? I have also tried logged in vagrant run apt-get update && apt-get update but seems it doesn't helped to me. Somewhere i have read that changing php version from 56 to 55 should make to work but i also doesn't solved the issue.

**Comment by viion** [comment]
I had the exact same issue and setting PHP to 5.5 fixed it, I cant get it working on 5.6

**Comment by jtreminio** [comment]
Sounds like redis module is not available for 32-bit under php 5.6

Not a puphpet issue - it's an upstream issue.

Sorry guys!
